nixos/modules/services/networking/fastnetmon-advanced.nix

```diff
@@ -0,0 +1,205 @@
+{ config, lib, pkgs, ... }:
+
+let
+  cfg = config.services.fastnetmon-advanced;
+  settingsFormat = pkgs.formats.yaml { };
+
+  # obtain the default configs by starting up ferretdb and fcli in a derivation
+  default_configs = pkgs.runCommand "default-configs" {
```

Comment:
You might want to add here, that FNM stores its configuration in the database and give some more context here.
You mentioned it in the PR but it is valuable information for maintaining this beast.